core/src/main/scala/kafka/controller/KafkaController.scala

```diff
@@ -19,28 +19,27 @@ package kafka.controller
 import java.util.concurrent.TimeUnit
 
 import com.yammer.metrics.core.Gauge
-import kafka.admin.{AdminUtils, PreferredReplicaLeaderElectionCommand}
+import kafka.admin.AdminOperationException
 import kafka.api._
 import kafka.cluster.Broker
 import kafka.common._
-import kafka.log.LogConfig
 import kafka.metrics.{KafkaMetricsGroup, KafkaTimer}
 import kafka.server._
-import kafka.utils.ZkUtils._
 import kafka.utils._
-import org.I0Itec.zkclient.exception.{ZkNoNodeException, ZkNodeExistsException}
-import org.I0Itec.zkclient.{IZkChildListener, IZkDataListener, IZkStateListener}
+import org.I0Itec.zkclient.IZkStateListener
```

Comment:
unused import org.I0Itec.zkclient.IZkStateListener and org.apache.zookeeper.Watcher.Event.KeeperState